packages/maskbook/src/plugins/Voicechat/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface VoicechatSetting {
+    active: boolean
```

Comment:
I suggest not add this kind of property because if the meta exists, the voice chat must be opened. Instead, I suggest allowing custom server URL and chat room name, etc...

Reply:
Yes, true. I just added an option to provide a custom server URL 🙂

I didn't add option for chat room name because in my opinion it doesn't really make sense to have one when each voice-chat room is already tied to a tweet.